package.json

```diff
@@ -20,14 +20,14 @@
   },
   "scripts": {
     "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
-    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
```

Comment:
Why this change?

Reply:
This is a mistake in the merge, sorry.

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad